clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -393,6 +393,14 @@ def warn_hlsl_langstd_minimal :
           "recommend using %1 instead">,
   InGroup<HLSLDXCCompat>;
 
+def err_hlsl_semantic_index_overlap : Error<"semantic index overlap %0">;
+
+def err_hlsl_semantic_missing : Error<"semantic annotations must be present "
```

Comment:
So I see new diagnostic but no tests that seem to exercise them? This does for every diagnostic not tested.

Reply:
Added a test for the missing semantic, and removed the 2 others which should be added in a later PR.